server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/GroupMappingTest.java

```diff
@@ -288,11 +289,11 @@ public void messageShouldNotBeSentWhenGroupLoopMapping() throws Exception {
         webAdminApi.put(GroupsRoutes.ROOT_PATH + "/" + GROUP_ON_DOMAIN2 + "/" + GROUP_ON_DOMAIN1);
 
         messageSender.connect(LOCALHOST_IP, SMTP_PORT)
-            .sendMessage(FakeMail.builder()
+            .sendEnvelope(FakeMail.builder()
```

Comment:
I don't see why this sendEnvelope is useless, sendMessage should trigger the error as well

Reply:
No. Send message send an additional DATA command which override the return code I wonna check!